app/components/Checkbox.js

```diff
@@ -7,12 +7,21 @@ const styles = {
     padding: '2px 0',
     margin: '5px 0',
     fontSize: 20,
-    cursor: 'pointer',
-    textAlign: 'center',
-    transition: 'background-color .2s ease-in-out'
+    cursor: 'pointer'
+  },
+  checkboxStatus: {
+    composes: 'glyphicon',
+    color: '#155799',
```

Comment:
Could we move this globally used colors to somewhere like `app/helpers/colors/index.js` and export it from `app/helpers` as `{ colors }`? This way we would make it more DRY.

I can do it myself if you agree :)